schemata/text.yaml

```diff
@@ -9,7 +9,15 @@ properties:
   id:
     "$ref": "base.json#/definitions/uuid"
   type:
-    "$ref": "contentblock.json#/definitions/type"
+    description: Block type
+    example: "text"
+    type: string
+    enum:
+      - text
+      - code
+      - quote
+      - hr
+      - unknown
```

Comment:
Unknown is called `unknown` because it is a block type html-flatten doesn't support. We know it has the minimum information available, but don't know how it should be rendered/enriched